modin/pandas/test/dataframe/test_iter.py

```diff
@@ -258,6 +259,24 @@ def test___setattr__():
     df_equals(modin_df, pandas_df)
 
```

Comment:
```suggestion
        # While `new_col` is not a column of the dataframe,
        # it should be accessible with __getattr__.
        assert modin_df.new_col.equals(pandas_df.new_col)
```

This is not strictly part of the bug, but we should check that we're setting non-column attributes correctly after your fix.

Reply:
I don't think `equals` would work here since these are list types, so I just used the `==` operator which should work fine for these lists I think.